projects/1.12.2/assets/elementaristics/patchouli_books/liber_elementium/zh_cn/entries/energy/entry_combustion.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "神秘燃烧产能室",
+  "icon": "elementaristics:generator_combustion",
+  "category": "energy",
+  "advancement": "elementaristics:progress_energy",
+  "pages": [
+    {
+      "type": "text",
+      "text": "$(item)神秘燃烧产能室$() 使用 $(item)火之元素$()产生OE。$(br)每个火之元素的可以燃烧100ticks，产能速率为 10 OE/t。$(br)如果没有其它的机器消耗它的能量，它会自动停止工作。"
```

```suggestion
      "text": "$(item)神秘燃烧产能室$() 使用 $(item)火之元素$()产生OE。$(br)每个火之元素的可以燃烧 100 刻，产能速率为 10 OE/刻。$(br)如果没有其它的机器消耗它的能量，它会自动停止工作。"
```
你这个提交量过于离谱（X
不过好在上古天坑的精卫又回来了（）

---

艹